.github/ISSUE_TEMPLATE/project-submission.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Project Submission
+about: Submit your project to the Silk.NET Community
+title: 'Project Submission: '
+labels: project submission
+assignees: Perksey, ThomasMiz, HurricanKai
+
+---
+
+## Basic Information
+
+<!-- Please fill out the below items -->
+
+- Project Name: 
+- Current Maintainers:
+- Current License: 
+- Used Silk.NET Libraries:
```

Comment:
Is this for people to put, for example, "Windowing, OpenGL, Input"?
I'd add something to clarify and change it to "Silk.NET Libraries used: " which sounds clearer to me

Reply:
```suggestion
- Used Silk.NET Libraries (e.g. Windowing, Input, OpenGL, Vulkan, etc):
```